deployments/ansible/roles/signalfx-agent/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# signalfx-agent handler
+- name: restart signalfx-agent
```

Comment:
How does this get used?  Does it just automatically restart the agent every time ansible is run or only on certain conditions?  Ideally the agent should only restart when its config file changes and not otherwise.